### PR TITLE
Add more information on rustdoc search

### LINKED ISSUE #65735: Add doc tip with '?' for exact search (within double quotes)
Currently, it's not specified into the help popup that you can make a search with double quotes. Also, might be interesting to maybe improve the text inside this helper.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #65735.

r? @kinnison

### CODE REVIEW

File: src/librustdoc/html/layout.rs [hunk 1]
```diff
@@ -150,6 +150,13 @@ pub fn render<T: Print, S: Print>(
                     Search multiple things at once by splitting your query with comma (e.g., \
                     <code>str,u8</code> or <code>String,struct:Vec,test</code>)\
                 </p>\
+                <p>\
+                    You can look for items with an exact name by putting double quotes around \
+                    your request: <code>\"string\"</code>\
+                </p>\
+                <p>\
+                    Look for items inside another one by searching a path: <code>vec::Vec</code>\
```
Comment (kinnison) [inline src/librustdoc/html/layout.rs]:
```suggestion
                    Look for items inside another one by searching for a path: <code>vec::Vec</code>\
```

**Review by kinnison — APPROVED** [review]
One tiny (optional) tweak, otherwise :+1:

**Comment by GuillaumeGomez** [comment]
Updated.

**Review by kinnison — APPROVED** [review]
:+1:

**Comment by JohnTitor** [comment]
r? @Dylan-DPC 
or someone from rustdoc team

**Pull request merged**